tests/models/mbart50/test_tokenization_mbart50.py

```diff
@@ -304,3 +304,11 @@ def test_tokenizer_translation(self):
                 "forced_bos_token_id": 250001,
             },
         )
+
+    def test_inconsistent_decoding(self):
+        text = "<s>[special_token] normal text"
```

Comment:
I don't think this tests special tokens at all - writing `[special_token]` doesn't make it a special token.

Reply:
Please now check if it's ok.

Reply:
The space is eaten between `special` and `<unk>` which I don't believe is intended. You also are not testing for the two cases as requested by @ArthurZucker in [his comment](https://github.com/huggingface/transformers/pull/28424#discussion_r1448968133).